### PR TITLE
fix(core): fix app boot speed

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
Current dependencies on/for this PR:
* **#5885** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5885?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5885?utm_source=stack-comment).

**Review by Brooooooklyn — APPROVED** [review]

File: packages/frontend/core/src/providers/modal-provider.tsx [hunk 1]
```diff
@@ -216,7 +216,9 @@ export const SignOutConfirmModal = () => {
 
   const onConfirm = useAsyncCallback(async () => {
     setOpen(false);
-    await signOutCloud();
+    await signOutCloud({
+      redirect: false,
+    });
```
Comment (pengx17) [inline packages/frontend/core/src/providers/modal-provider.tsx:221]: is this relevant?

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/providers/modal-provider.tsx (hunk 1 above)
Comment (EYHN) [reply packages/frontend/core/src/providers/modal-provider.tsx:221]: There is jump logic below this line, If redirected here, the code below will not be run.

Comment (EYHN) [reply packages/frontend/core/src/providers/modal-provider.tsx:221]: The previously added `workspaceListStatus.loading` judgment is to solve the page not found error of jumping to cloud workspace after sign out.

**Pull request merged**